openmc/mesh.py

```diff
@@ -193,6 +193,78 @@ def centroids(self):
         s1 = (slice(1, None),)*ndim + (slice(None),)
         return (vertices[s0] + vertices[s1]) / 2
 
+    def write_data_to_vtk(self, points, filename, datasets, volume_normalization=True):
+        """Creates a VTK object of the mesh
+
+        Parameters
+        ----------
+        points : list or np.array
+            List of (X,Y,Y) tuples.
+        filename : str
+            Name of the VTK file to write.
+        datasets : dict
+            Dictionary whose keys are the data labels
+            and values are the data sets.
+        volume_normalization : bool, optional
+            Whether or not to normalize the data by
+            the volume of the mesh elements.
+            Defaults to True.
```

Comment:
The default shows up in the argument list so probably not necessary to list it explicitly here
```suggestion
```